src/Microsoft.DotNet.Web.ProjectTemplates/RazorClassLibrary-CSharp.csproj.in

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
FYI: We will always hardcode to this, but we still define frameworks in `template.json` to keep VS happy so that we get template highlighted when .NET Framwork  or NETCoreApp2.1 is selected in VS